src/plots/plot_processing.jl

```diff
@@ -0,0 +1,110 @@
+"Modify distribution grid data dictionary"
+function distr_data(case::Dict{String,<:Any})
+    # create new dict
+    case_dist = Dict{String, Any}("bus"=>Dict{String, Any}(),
+        "gen"=>Dict{String, Any}(),
+        "branch"=>Dict{String, Any}(),
+        "switch"=>Dict{String, Any}(),
+        "transformer"=>Dict{String, Any}()
+        )
```

Comment:
This does not copy the metadata of the grid, like the name, per unit etc.  I would make a deep-copy of the old network and replace these component dictionaries with empty ones
```julia
case_dist = deepcopy(case)
for comp_type in ["gen","branch","bus","switch","transformer"]
    case_dist[comp_type] = Dict{String,Any}()
end
```